scripts/ci/libraries/_initialization.sh

```diff
@@ -131,7 +131,7 @@ function initialize_common_environment {
         print_info
         print_info "Mount whole airflow source directory for static checks"
         print_info
-        EXTRA_DOCKER_FLAGS=( \
+        EXTRA_DOCKER_FLAGS=(
             "-v" "${AIRFLOW_SOURCES}:/opt/airflow" \
```

Comment:
```suggestion
            "-v" "${AIRFLOW_SOURCES}:/opt/airflow"
```
Also redundant? (no backslash in L:144)

Reply:
Yep